content/docs/user-guide/profiling/cpu_profiling/_index.md

```diff
@@ -0,0 +1,423 @@
+
+![Imgui profiler overview](/images/user-guide/profiling/imgui/cpu-profiler-open.png)
+
+### Launch a capture
+
+Upon being opened the profiler will be showing the performance for last few frames. Click on the **"Resume/Pause"** button in the top left to toggle the update.
+
+If you want to capture more than a few frames, press the **"Begin"** button and let the profiler run, maybe jump into gameplay to perform specific actions. Once that you have enough data, you can press the **"End"** button to stop the capture and be able to navigate it.
+
+![Imgui profiler overview](/images/user-guide/profiling/imgui/overview.png)
+
+### Navigate your capture
+
+The navigation within the profiler is done using the mouse. You can click on the **"Swap to visualizer/statistics"** to switch between the two primary views of the Profiler. Clicking on an item in the visualizer view will switch to the statistics view, as seen below.
+
+![Imgui profiler statistics](/images/user-guide/profiling/imgui/start.png)
+
+Use the **Find Region** in the visualizer to search for specific events. The timeline is interactive, click on it with the left mouse button to have it focused, then :
+
+- Hold left mouse button to move around the timeline
+- Hold the ctrl key and use the mouse wheel to change the timeline scale
+
+![Imgui profiler search](/images/user-guide/profiling/imgui/profile-event-search.jpg)
+
+When the timeline is scaled-out, you will be able to see the frametime in ms for every captured frame in the profiler. This is an easy way to find frames which are over your budget.
+
+### Save your capture
+
+Press the "Capture" button to save your captured data in a json file. The path will be indicated right above the button. You can load any previous capture thanks to the "Load file" button.
+
+![Imgui profiler save](/images/user-guide/profiling/imgui/save.png)
+
+## Prerequisites for Tracy, Optick and Superluminal
+
+These profiler Gems are located in the [o3de-extras](https://github.com/o3de/o3de-extras) github repository. 
+
+1. This repository follows the Remote Content format, you can follow [this documentation](/docs/user-guide/remote-content/use-a-remote-repository) to learn how to **register a remote repository**.
+
+2. Else if you prefer to download o3de-extra yourself, you can follow [this documentation](/docs/user-guide/project-config/register-gems). Within o3de-extra folder, the profiler gems are located in "*Gems/ExternalProfilers*".
+
+Once that the profiler gem that you want to use has been registered, you need to enable it for your project and rebuild the project. Refer to the [first steps of enabling the Built-In profiler](#enabling-the-built-in-profiler-gem) to see how to enable a gem for your project.
+
+## Using Tracy profiler (All platforms)
```

Comment:
From this section down, how much of this is already covered by the docs for the respective profilers? It's a general rule of thumb to not document the general usage of third party applications. What we document here should only be where the profilers intersect with O3DE.

Reply:
The issue is that the documentation for these tools is either nonexistant or too obtuse for newcommer, I try to give broad strokes on how to launch them and get quick result, but feel free to pinpoint an area if you think it is too detailed

Reply:
If you take tracy for example, this is the documentation pdf, I don't expect a lot of people do dig into it https://github.com/wolfpld/tracy/releases/latest/download/tracy.pdf

Reply:
Alright, yeah, that's what I kind of expected. This would be one of the exceptions to the rule of thumb, then. I'll get it reviewed and edited shortly.